Documentation/gettingstarted/k8s-install-gke.rst

```diff
@@ -70,23 +70,23 @@ Deploy Cilium release via Helm:
 
     kubectl create namespace cilium
     helm install cilium |CHART_RELEASE| \\
-      --namespace cilium \\
+      --namespace kube-system \\
       --set global.nodeinit.enabled=true \\
       --set nodeinit.reconfigureKubelet=true \\
       --set nodeinit.removeCbrBridge=true \\
-      --set global.cni.binPath=/home/kubernetes/bin
+      --set global.cni.binPath=/home/kubernetes/bin \\
+      --set global.restartPods
```

Comment:
This is intentionally omitted by default as restarting pods is a bit "aggressive". Maybe moving the note below before the `helm install ...` instruction rather than after would be a solution?

Reply:
Ah, okay, I understand. We're still asking users to restart all pods in the next section as part of the quick start guide though. Should we rework that next section? Is there a way to avoid the restarts?

Reply:
I've reworded a bit to try and clarify why `global.restartPods` is not set by default and moved that note before the actual command.